routes/html/index.js

```diff
@@ -5,35 +5,38 @@ const sequelize = require("../../config/connection");
 const router = require("express").Router();
 
 router.get("/", (req, res) => {
-  if (req.user) {
+  const loggedIn = req.user;
```

Comment:
`!!req.user`

Reply:
router.get("/", withAuth, async (req, res) => {
  const loggedIn = req.user;
  if (!!loggedIn) {
    // render the dashboard
    const fetchName = req.user.firstName;
    const fetchUsername = req.user.username;
    res.render("dashboard", { fetchName, fetchUsername, loggedIn });
  } else {
    // render the landing page
    res.render("landingpage");
  }
});

Reply:
will be changed

```
router.get("/", withAuth, async (req, res) => {
  const loggedIn = req.user;
  if (!!loggedIn) {
```